platforms/Windows/cli/cli.wxs

```diff
@@ -270,6 +719,15 @@
 
       <ComponentGroupRef Id="collections" />
       <ComponentGroupRef Id="llbuild" />
+      <ComponentGroupRef Id="SwiftBuild" />
+      <ComponentGroupRef Id="SWBAndroidPlatformResources" />
+      <ComponentGroupRef Id="SWBApplePlatformResources" />
+      <ComponentGroupRef Id="SWBCoreResources" />
+      <ComponentGroupRef Id="SWBGenericUnixPlatformResources" />
+      <ComponentGroupRef Id="SWBQNXPlatformResources" />
+      <ComponentGroupRef Id="SWBUniversalPlatformResources" />
+      <ComponentGroupRef Id="SWBWebAssemblyPlatformResources" />
+      <ComponentGroupRef Id="SWBWindowsPlatformResources" />
```

Comment:
I think that SwiftBuild should be its own ComponentGroup. Is there a plan to merge SwiftBuild into SPM (not as a dependency but repository-wise)?